bubble-wrap-style.yaml

```diff
@@ -1382,11 +1446,96 @@ layers:
             text-blend-order:
                 text_source: global.ux_language_text_source
                 visible: false    # labels are enabled by each layer below
+                priority: |
+                    function() { return (feature['min_zoom'] + (1 - 1 / feature['collision_rank']) * 0.1) || 65; }
```

Comment:
Use the v1.7 `collision_rank` for predictable POI labeling. 

Really we should also have a server budget and do some collisions there for low- and mid-zooms.